src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -44,6 +44,7 @@
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
 	public static final String AUDIO_TRANSCODE = "audio/transcode";
 	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AAC_TYPEMIME = "audio/mp4";
```

Comment:
Not in the DLNA standard, though some renderer will use that one.
That why it is noted in the DefaultRenderer documentation and can be modified by user depending on their renderer.
I consider that the default MIME type should be the DLNA standard first.